components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1162,6 +1225,36 @@ void triggerPreRevokeListeners(OAuthRevocationRequestDTO
         }
     }
 
+    void triggerBulkPostRevokeListeners(OAuthRevocationResponseDTO revokeRespDTO, List<AccessTokenDO> accessTokenDOs) {
+
+        OAuthEventInterceptor oAuthEventInterceptorProxy = OAuthComponentServiceHolder.getInstance()
+                .getOAuthEventInterceptorProxy();
+        if (oAuthEventInterceptorProxy != null && oAuthEventInterceptorProxy.isEnabled()) {
+            try {
+                Map<String, Object> paramMap = new HashMap<>();
```

Comment:
Move this out of the try block please.

Reply:
Addressed in [18e795](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/18e7958d172592cf2743ae36381a7fbecd2b74e0)